packages/modules/src/firebase/functions-checkout/new-order.ts

```diff
@@ -0,0 +1,44 @@
+import type { Orders } from '@cloudcommerce/types';
+import { logger } from 'firebase-functions';
+import api from '@cloudcommerce/api';
+
+type BodyOrder = Omit<Orders, '_id' | 'created_at' | 'updated_at' | 'store_id' >
+
+export default async (
+  orderBody:BodyOrder,
+  acessToken:string,
+) => {
+  try {
+    const orderId = (await api.post(
+      'orders',
+      orderBody,
+      {
+        headers: {
+          Authentication: `Bearer ${acessToken}`,
+        },
```

Comment:
```suggestion
        acessToken,
```

https://github.com/ecomplus/cloud-commerce/blob/main/packages/api/src/types.ts#L53
https://github.com/ecomplus/cloud-commerce/blob/main/packages/api/src/api.ts#L44-L47

:wink:

Reply:
Lembrando que esse `accessToken` é o do customer que deve vir pronto no cabeçalho `Authorization` na request do checkout, e deve ser usado em todas (não só essa) as requests pra Store API durante o checkout